### DISCUSSION TITLE
Search doesn't support count and doesn't accept <C-g> <C-t>

### DISCUSSION THREAD

**Discussion by Oliveiras** [JetBrains/ideavim#584] [Ideas]
Suppose the cursor is in the beginning of the following line:
`one two two three`

On normal vim, I can search for the term "two" and stop at the second occurrence using `2/two<CR>` or `/two<C-g><CR>`.

None of this works in IdeaVim.

This is very useful when you are not simply jumping, but applying a command (like yanking or deleting) or recording a macro.

**Comment by AlexPl292** [comment]
Thank you for the report. We don't have any immediate plans regarding it, but you can track the state of the issue here: https://youtrack.jetbrains.com/issue/VIM-2836/Search-doesnt-work-with-count
